src/CMakeLists.txt

```diff
@@ -1,15 +1,11 @@
 include(GetGitRevisionDescription)
 
 enable_language(C ASM)
-set(bindir ${CMAKE_INSTALL_PREFIX}/bin)
-set(sbindir ${CMAKE_INSTALL_PREFIX}/sbin)
-set(libdir ${CMAKE_INSTALL_PREFIX}/lib)
-set(sysconfdir ${CMAKE_INSTALL_PREFIX}/etc)
-set(pkgdatadir ${CMAKE_INSTALL_PREFIX}/share)
```

Comment:
please do not remove them at this moment, we use these variables for substitution in our .in files, for example, `src/ceph-crush-location.in` and `src/ceph-debugpack.in`. maybe we will clean them up in master by using the cmake variables. but not in jewel, where we just do the bug fixes.